.github/workflows/publish-on-pypi.yml

```diff
@@ -6,17 +6,52 @@ on:
     types: [published]
 
 jobs:
-  build-and-upload:
+  build-wheels:
+    strategy:
+      matrix:
+        os: [windows-latest, macos-latest, ubuntu-latest]
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+
+    - name: Check out the release commit
+      uses: actions/checkout@v3
+
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v2
+      with:
+        platforms: arm64
+      if: runner.os == 'Linux'
+
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3.11'
+
+    - name: Build wheels
+      uses: pypa/cibuildwheel@v2.12.1
+
+    - name: Check and upload wheels
+      env:
+        TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
+        TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
+      run: |
+        python -m pip install twine
+        python -m twine check --strict wheelhouse/*.whl
+        python -m twine upload wheelhouse/*.whl
+
+  build-sdist:
     runs-on: ubuntu-latest
 
     steps:
     - name: Check out the release commit
-      uses: actions/checkout@v2
+      uses: actions/checkout@v3
 
     - name: Set up Python
-      uses: actions/setup-python@v2
+      uses: actions/setup-python@v4
       with:
-        python-version: '3.8'
+        python-version: '3.11'
```

Comment:
Any reason why the sdist should be created with the latest possible version of Python? I would assume an older version was better (and Cython source would be compiled by the user of the sdist)

Reply:
No particular reason other than the assumption that most recent python would automatically get the most current packaging infrastructure.  Cython shouldn't be an issue because I don't think we ship the generated .cpp files in the sdist any more.

Reply:
Looks like I was wrong about that: https://github.com/enthought/enable/blob/2230bb4487d292f91c724b34e5eb02abc193a60b/MANIFEST.in#L17

Probably should remove that - the isolated builds should work install Cython correctly from the sdist if needed.

And I also spot a stray gl reference in the manifest as well.